ocs_ci/ocs/constants.py

```diff
@@ -1338,7 +1338,7 @@
     "Brown": ["/z_cluster/"],
     "Green": ["/pv_services/", "/storageclass/"],
     "Blue": ["/monitoring/"],
-    "Red": ["/mcg/", "/rgw/"],
+    "Red": ["/mcg/", "/rgw/", "/system/"],
```

Comment:
@vavuthu E2E came up with system tests recently from 4.9. The responsibility of analysing this test failure belongs to Magenta squad. Please move it under magenta

Reply:
done

Reply:
@PrasadDesala coud you update the "Tests Ownership (squads)" spreadsheet

Reply:
Done